src/main/java/nextstep/courses/infrastructure/JdbcStudentRepository.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.courses.infrastructure;
+
+import nextstep.courses.domain.session.StudentRepository;
+import nextstep.courses.domain.session.entity.StudentEntity;
+import org.springframework.jdbc.core.JdbcOperations;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+@Repository("studentRepository")
+public class JdbcStudentRepository implements StudentRepository {
+    private final static StudentRowMapper STUDENT_ROW_MAPPER = new StudentRowMapper();
+    private JdbcOperations jdbcTemplate;
```

Comment:
키워드 순서에도 [관례](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)가 있으니 참고해 주세요 
```suggestion
    private static final StudentRowMapper STUDENT_ROW_MAPPER = new StudentRowMapper();
    private final JdbcOperations jdbcTemplate;
```